lib/main.coffee

```diff
@@ -29,7 +39,7 @@ module.exports =
 
     # Check version of coala
     version = helpers.exec(@executable, ['--version']).then (result) ->
-      if result <= COALA_MIN_VERSION
+      if cmpVersions(result, COALA_MIN_VERSION) < 0
```

Comment:
so the python way of doing this would be `tuple(int(num) for num in '1.2.3'.split('.'))` (which could e put into a simple function) and then use the tuple `>` operation which would compare first elements first, then second etc.; it behaves very very well for versions:

```
>>> (1,) > (1,2)
False
>>> (1,2,3) > (1,2)
True
```

is such a thing possible for coffeescript?